### PR TITLE
Fixed #33721 -- Added DE-9IM link in GEOS docs.

### PR DESCRIPTION
**Pull request by Ksenofanex**
Issue: https://code.djangoproject.com/ticket/33721

### CODE REVIEW

File: docs/ref/contrib/gis/geos.txt [hunk 1]
```diff
@@ -410,7 +410,7 @@ return a boolean.
     The ``covers`` predicate has the following equivalent definitions:
 
     * Every point of the other geometry is a point of this geometry.
-    * The DE-9IM Intersection Matrix for the two geometries is
+    * `The DE-9IM Intersection Matrix <https://en.wikipedia.org/wiki/DE-9IM>`_ for the two geometries is
```
Comment (felixxm) [inline docs/ref/contrib/gis/geos.txt:413]: I'd move it to the named link, e.g.
```diff
diff --git a/docs/ref/contrib/gis/geos.txt b/docs/ref/contrib/gis/geos.txt
index a8057dc631..08a64a2a1d 100644
--- a/docs/ref/contrib/gis/geos.txt
+++ b/docs/ref/contrib/gis/geos.txt
@@ -410,7 +410,7 @@ return a boolean.
     The ``covers`` predicate has the following equivalent definitions:
 
     * Every point of the other geometry is a point of this geometry.
-    * The DE-9IM Intersection Matrix for the two geometries is
+    * The `DE-9IM`_ Intersection Matrix for the two geometries is
       ``T*****FF*``, ``*T****FF*``, ``***T**FF*``, or ``****T*FF*``.
 
     If either geometry is empty, returns ``False``.
@@ -423,6 +423,8 @@ return a boolean.
     added benefit, ``covers()`` is more amenable to optimization and hence
     should outperform :meth:`~GEOSGeometry.contains`.
 
+    .. _DE-9IM: https://en.wikipedia.org/wiki/DE-9IM
+
 .. method:: GEOSGeometry.crosses(other)
 
     Returns ``True`` if the DE-9IM intersection matrix for the two Geometries
```

**Review by felixxm — COMMENTED** [review]
@Ksenofanex Thanks :+1:

**Comment by felixxm** [comment]
@Ksenofanex Thanks :+1: Welcome aboard :boat:

**Comment by Ksenofanex** [comment]
@felixxm The pleasure is mine. :) Hope it'll be the first of many.

**Comment by felixxm** [comment]
buildbot, test this please.

**Pull request merged**